docs/docs/04-multiple-components.md

```diff
@@ -124,7 +124,7 @@ The situation gets more complicated when the children are shuffled around (as in
 
 When React reconciles the keyed children, it will ensure that any child with `key` will be reordered (instead of clobbered) or destroyed (instead of reused).
 
-The `key` should *always* be supplied directly to the components in the array, not to the container HTML child of each component in the array:
+The `key` should *always* be supplied directly to the elements or components in the array, not to the container HTML child of each component in the array:
```

Comment:
> HTML child

That's also pretty wrong and confusingly worded. Want to try to clean the rest of this up as well?